### DISCUSSION TITLE
My TOC not work

### DISCUSSION THREAD

**Discussion by yeshadoo** [hugo-fixit/FixIt#358] [Q&A]
i have copy config.toml from manual, but it cannot show on post,
anyone meet this problem? thanks.

**Comment by yeshadoo** [comment]
![Uploading image.png…]()

**Reply by Lruihao** [reply]
Oh no, it looks like the image you provided was 404 status.

**Comment by salvari** [comment]
Maybe not related with your problem, but I've lost the lateral TOC when using Firefox with "I don't care about cookies" extension.
You need to accept cookies if you want lateral TOC working properly.

Static TOC (changing parameters.toml) seems to work fine even without cookies.
